Platforms/OSX/AppKit/NSApplication.swift

```diff
@@ -710,7 +710,7 @@ enum NSApplicationDelegateReply : UInt {
 }
 extension NSApplication {
   var windowsMenu: NSMenu?
-  func arrangeInFront(sender: AnyObject?)
+  func arrangeIn(front sender: AnyObject?)
```

Comment:
Ouch. We might have to special-case things that look like actions (presumably, Void methods with a single `sender: AnyObject?` parameter in subclasses of `NSResponder` or `UIResponder`).